### PR TITLE
Update latest equistore version

### PR DESCRIPTION
**Pull request by PicoCentauri**
In [PR147](https://github.com/lab-cosmo/equistore/pull/147)of equistore the meta data checks where moved to a different location. This PRs updated this. Due to our fixed version policy I had to introduce an extra branch on rascaline to keep up with these changes.

Also I update the operations import and update the test folders to be same name structure as on equistore and rascaline.

----
:books: Documentation preview :books:: https://equisolve--44.org.readthedocs.build/en/44/

### CODE REVIEW

**Comment by PicoCentauri** [comment]
I don't know why pytest is not finding anything in the CI...

```
rootdir: /home/runner/work/equisolve/equisolve, configfile: pyproject.toml, testpaths: tests/equisolve_tests/*.py, tests/equisolve_tests/**/*.py
[21](https://github.com/lab-cosmo/equisolve/actions/runs/4480577764/jobs/7876083270?pr=44#step:5:22)
collected 0 items
```

An idea @agoscinski or @Luthaf ?

File: pyproject.toml [hunk 1]
```diff
@@ -17,6 +17,8 @@ lines_after_imports = 2
 known_first_party = "equisolve"
 
 [tool.pytest.ini_options]
-addopts = "-p pytest_cov"
-python_files = ["test_*.py", "*_test.py", "tests.py"]
-testpaths = "tests"
+python_files = ["*.py"]
+testpaths = [
+    "tests/equisolve_tests/*.py",
+    "tests/equisolve_tests/**/*.py",
```
Comment (Luthaf) [inline pyproject.toml:23]: Why are you placing tests here and not just as `tests/*.py`?

**Comment by Luthaf** [comment]
The Python 3.6 builder seems to get an older version of tox (3.28), while the other builders are getting 4.4. Maybe it is time to drop 3.6 =)

File: pyproject.toml (hunk 1 above)
Comment (PicoCentauri) [reply pyproject.toml:23]: I tried things and hoped that this changes something.

**Comment by PicoCentauri** [comment]
Dropping versions solved all our problems 😬

**Pull request merged**